### PR TITLE
expense tracker app running issue fixed

### LINKED ISSUE #4143: [BUG]: Issue for running Expense Tracker Flutter Application
**Describe the bug**
File Structure of Expense Tracker in Flutter Project directory is not proper and the application is not running . 

**Expected behavior**
Want to fix this issue and improve the application and want to add some features to it.

### PR DESCRIPTION
**Pull request by aritroo**
# Title and Issue number 
Expense Tracker

Close #4143 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [x] I have fixed the file structure and the app not running issue
- [x] I have added addtional features and the UI is improved
- [x] I have updated the readme file with the screenshots.

### CODE REVIEW

**Comment by aritroo** [comment]
Hello @MohitGupta121 , Sir please review my PR and also add a label for gssoc'23

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
Remove this

File: Flutter Projects/.DS_Store
Comment (MohitGupta121) [inline Flutter Projects/.DS_Store:1]: Remove this also

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by aritroo** [comment]
can't able to make changes and I am closing this PR and I will make the desired changes in new PR.

**Pull request closed without merging**